src/core_write.cpp

```diff
@@ -209,6 +215,11 @@ void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry,
                 txinwitness.push_back(HexStr(item));
             }
             in.pushKV("txinwitness", txinwitness);
+            if (txundo) {
+                const CTxOut& txout = txundo->vprevout[i].out;
+                // Suppress fuzz test failures by ignoring nonsense values
```

Comment:
It seems strange to me to have code in the produce that exists only to work around shortcomings in the fuzz tests. Which fuzz test fails if you don't do this? (the only fuzz tester that calls `TxToUniv()` doesn't provide a `txundo` param).

Reply:
Agree with @jnewbery . It is impossible to hit this condition in production, so an assert or just nothing at all might be more appropriate.

Reply:
I suggest stating the pre-condition in the form of an assertion or documentation. Then it is the fuzzer's responsibility the fulfil all explicitly stated pre-conditions.